3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_RPI.yaml

```diff
@@ -0,0 +1,379 @@
+        - 1262.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 20.2 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 905.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 50.1 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 952.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 53.1 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1017.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 53.2 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1039.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 50.9 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1042.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 46.8 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1121.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 50.3 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1144.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 49.9 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1147.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 50.6 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1166.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 48.2 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+    - temperature:
+        - 1282.0 kelvin
+      ignition-delay:
+        - 47.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 46.6 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
+      
```

Comment:
Please make sure to end the file with a newline. Editors such as Atom and VSCode can be configured to add this automatically 😄